spec/query_spec.rb

```diff
@@ -892,6 +892,71 @@ def sql_select_numbers(select_string)
       @drill_through.rows.all?{|r| r.any?{|c| c}}.should be_true
     end
 
+    it "should return member name and property values" do
+      @drill_through = @result.drill_through(row: 0, column: 0,
+        return: [
+          "Name([Customers].[Name])",
+          "Property([Customers].[Name], 'Gender')"
+        ]
+      )
+      @drill_through.column_labels.should == [ "Name", "Gender" ]
+      @drill_through.rows.should == @sql.select_rows(<<-SQL
+        SELECT
+          customers.fullname,
+          customers.gender
+        FROM
+          sales,
+          customers,
+          time,
+          products,
+          product_classes
+        WHERE
+          time.quarter = 'Q1' AND
+          time.the_year = 2010 AND
+          product_classes.product_family = 'Drink' AND
+          products.product_class_id = product_classes.id AND
+          sales.product_id = products.id AND
+          sales.time_id = time.id AND
+          customers.id = sales.customer_id
+        ORDER BY
+          customers.fullname,
+          customers.gender
+      SQL
+      )
+    end
+
+    it "should group by" do
+      @drill_through = @result.drill_through(row: 0, column: 0,
+        return: [
+          "[Product].[Product Family]",
+          "[Measures].[Unit Sales]",
+          "[Measures].[Store Cost]"
+        ],
+        add_group_by: true
```

Comment:
It seems that `group_by` parameter would look better from the usage perspective. Please change in all places.